### DISCUSSION TITLE
Any guide for moving existing default wordpress to wordplate?

### DISCUSSION THREAD

**Discussion by aneesdev** [wordplate/wordplate#331] [Q&A]
Hi @vinkla, 

First of all, thank you so much for the great package. Always appreciate the hard work that you have put into it.

My question is that currently, I have the default WordPress site. any minimal guide on how to migrate it to wordplate?

**Comment by vinkla** [comment]
I'm glad you like it!

1. If you already have a theme, you can drag and drop it into the theme directory.
2. If you have installed any plugins, [you may installed them with Composer instead.](https://github.com/vinkla/wordplate#plugins)
